packages/react/src/Alert/Alert.test.tsx

```diff
@@ -8,9 +8,11 @@ describe('Alert', () => {
     const { container } = render(<Alert />)
 
     const component = container.querySelector(':only-child')
+    const icon = component?.querySelector('.amsterdam-alert__icon')
```

Comment:
Whenever possible use `screen.getByRole()`. 

Side note: Perhaps also a good reason the think about using `role="alert"` for the error severity.